pom.xml

```diff
@@ -418,7 +418,7 @@
 
     <properties>
         <!--repo versions of quick-start-guide-->
-        <identity.agent.sso.version>5.1.18</identity.agent.sso.version>
+        <identity.agent.sso.version>5.1.22</identity.agent.sso.version>
```

Comment:
Did you run the integration tests with higher sso agent version (5.1.22)?

Reply:
Does the product IS have a direct dependency on this repo?

Reply:
Finally, any built pack contains only the product-is pom's updated versions jar files. These samples should be able to use with all the upcoming releases. So without updating the version in product-is, we can't move forward.

Reply:
The reason is you have mentioned above that this PR should be merged after merging the PR https://github.com/wso2-extensions/identity-agent-sso/pull/46. Without passing the integration test you can't merge the PR https://github.com/wso2-extensions/identity-agent-sso/pull/46 otherwise product-is tests will fail once you update the version. If you are not updating the sso-agent version in product-is those features won't go the upcoming releases.